exercises/dnd-character/example.js

```diff
@@ -0,0 +1,36 @@
+export class Character {
+  constructor() {
+    this.strength = this.getAbilityScore();
+    this.dexterity = this.getAbilityScore();
+    this.constitution = this.getAbilityScore();
+    this.intelligence = this.getAbilityScore();
+    this.wisdom = this.getAbilityScore();
+    this.charisma = this.getAbilityScore();
+    this.hitpoints = 10 + ability_modifier(this.constitution);
+  }
+
+  getAbilityScore() {
```

Comment:
I think this would be better as _static_ method (class method) on `Character`.

```javascript
export class Character {
  static generateAbility() {
    ...
  }
}
```

Reply:
I can do that, but then I believe we would need to remove the second suite of tests (lines 82 - 92), or at least the test case on lines 83-86.

Reply:
Change it to:

```javascript
    test('ability within range', () => {
      expect(Character.generateAbility()).toBeLessThanOrEqual(18);
      expect(Character.generateAbility()).toBeGreaterThanOrEqual(3);
    });
```

FYI I picked `generateAbility` here, but feel free to use a _better_ name. I don't think the _`get`_ prefix is right, but `rollAbility` or AbilityScore might be ok.

Reply:
Wait, scratch that. I've made the appropriate changes. Changes incoming.